### ISSUE TITLE
Any clarification on the data sources?

### ISSUE THREAD

**Issue by ykzeng** [descarteslabs/DL-COVID-19#1]
Hi team,

First of all I wanna thank you for doing such a great analysis on huge scale in such a short time. The mobility data are gonna provide useful insights in containing the COVID-19 as well as any other epidemics in the future. However, I do want to understand more, especially, where did this data originally come from? It seems that I cannot find the specific data source anywhere in Github and the paper. I believe it would be very helpful for the readers to know where these mobility data actually come from to determine if there is any biases in the sampling process.

Once again, thanks for the great work and I will keep updated with any news from your team!

**Comment by alexk307** [comment]
Agreed. I find it dangerous to be making these claims without disclosing your data sources.
